bin/setup-radiopadre-virtualenv

```diff
@@ -106,45 +123,59 @@ for line in out.decode().split("\n"):
 else:
     bye("jupyter kernelspec list did not return the radiopadre kernel. This is a bug?")
 message("Linking kernel.js to {kerneldir}")
-os.symlink(PADRE_PATH + "/html/radiopadre-kernel.js", kerneldir + "/kernel.js")
+shutil.copy2(PADRE_PATH + "/html/radiopadre-kernel.js", kerneldir + "/kernel.js")
 
 # add link to radiopadre web components in notebook static dir
 notebook_padre_www = notebook_dir + "/notebook/static/radiopadre-www"
 padre_www = PADRE_PATH + "/html"
-message("making link from {notebook_padre_www} to {padre_www}")
-if not os.path.exists(notebook_padre_www):
+remove_installation(notebook_padre_www)
+if options.editable:
+    message("making link from {notebook_padre_www} to {padre_www}")
```

Comment:
missing formating (f-string)

Reply:
oh i see the message `func` handles this

Reply:
Yeah that was me inventing f-strings before there were f-strings...